packages/libxml2.rb

```diff
@@ -24,6 +24,7 @@ class Libxml2 < Package
 
   depends_on 'zlibpkg'
   depends_on 'readline'
+  depends_on 'gcc11'
```

Comment:
Since libxml2 is a core package, that means gcc11 is now in core too.  Can you please update `tools/core_packages.txt`?  That also means you can remove it from buildessential.  Try a fresh install and then run `crew list installed` to see if it matches.  Better yet - figure out a way to avoid gcc11 as a dependency. :)

Reply:
I also forgot to mention it needs to be added to core.rb.

Reply:
I'm going to do a fresh install, but my build environment script adds some packages, so I can not at the moment compare `crew list installed` to the core packages list...

Reply:
Why wouldn't you want to match your build environment?  That seems like a potential recipe for false positive tests.

Reply:
meaning that I also install: `util_linux` `coreutils` `psmisc` `bash` on my build environment, because for instance, I use `linux32` for the armv7l container...

Reply:
(not having to add more packages after I start the container speeds up builds.)

Reply:
Those have the potential to have a meaningful impact for testing packages.

Reply:
We're supposed to be reducing the size of core, not expanding it. :)